conda/recipes/kvikio/meta.yaml

```
@@ -28,6 +28,8 @@ requirements:
     - setuptools
     - cython >=0.29,<0.30
     - cudatoolkit {{ cuda_version }}.*
+    - cmake>=3.20.1
+    - scikit-build>=0.13.1
```

Minor suggestion. Would space out the version constraints. This is the norm in recipes

```suggestion
    - cmake >=3.20.1
    - scikit-build >=0.13.1
```